### PR TITLE
Tool menu option to start the Android remote server.

### PR DESCRIPTION
**Pull request by michaelrgb**

### CODE REVIEW

**Comment by michaelrgb** [comment]
Maybe we could move this button into a device-specific menu somewhere?

File: renderdocui/renderdocui.csproj [hunk 1]
```diff
@@ -627,6 +627,10 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>copy $(SolutionDir)scripts\android_remoteserver.bat $(TargetDir)
```
Comment (baldurk) [inline renderdocui/renderdocui.csproj]: I'll need to check if this works in my build scripts, as I run msbuild from a bash environment so 'copy' isn't available. MSBuild has built-in ways to copy files to the output directory though so I can fix it if needs be.

**Comment by michaelrgb** [comment]
I took out the Process.WaitForExit() so the user can still interact with the main GUI without closing the cmd window. 

I was intending for the user to close the cmd window first, but its opened in the correct directory, so its very useful for then typing subsequent android_capture commands.

File: renderdocui/Windows/MainWindow.cs [hunk 2]
```diff
@@ -1006,18 +1006,32 @@ private LiveCapture OnCaptureTrigger(string exe, string workingDir, string cmdLi
             if (!PromptCloseLog())
                 return null;
 
-            string logfile = m_Core.TempLogFilename(Path.GetFileNameWithoutExtension(exe));
-
-            UInt32 ret = m_Core.Renderer.ExecuteAndInject(exe, workingDir, cmdLine, env, logfile, opts);
+            UInt32 remoteIdent;
+            if (m_Core.Renderer.Remote != null && m_Core.Renderer.Remote.Hostname.StartsWith("adb:"))
```
Comment (baldurk) [inline renderdocui/Windows/MainWindow.cs]: I think this is better placed to go in the remote server on the C++ side. We already know if we are connecting to android in `RENDERDOC_CreateRemoteServerConnection` which can be passed through creation, and the code can do something different inside `RemoteServer::ExecuteAndInject`.

Comment (michaelrgb) [inline renderdocui/Windows/MainWindow.cs]: Ok I think I can execute those shell commands from android native code in ExecuteAndInject.

Comment (michaelrgb) [inline renderdocui/Windows/MainWindow.cs]: Lets just merge the remoteserver command when youre ready, whilst I figure out the other half.

**Pull request merged**